### PR TITLE
Adjust condition for resolve-violations command, if ids are passed.

### PR DESCRIPTION
**Pull request by mohabusama**

### CODE REVIEW

File: fullstop/cli.py [hunk 1]
```diff
@@ -282,8 +282,8 @@ def resolve_violations(config, comment, since, region, meta, remeta, limit, viol
 
     kwargs['accounts'] = kwargs.get('accounts') or config.get('accounts')
 
-    if not kwargs['accounts'] and not kwargs['type'] and not region:
-        raise click.UsageError('At least one of --accounts, --type or --region must be specified')
+    if all([not violation_ids, not kwargs['accounts'], not kwargs['type'], not region]):
+        raise click.UsageError('At least one of --accounts, --type, --region or --violation-ids must be specified')
```
Comment (harti2006) [inline fullstop/cli.py:286]: does it also fail if I did `fullstop configure` before and specified a set of accounts?

Comment (mohabusama) [inline fullstop/cli.py:286]: If accounts is configured, then it won't fail, as `kwargs['accounts']` will have a value.

**Comment by harti2006** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**